data/human/culture conversations.txt

```diff
@@ -120,6 +120,35 @@ mission "Paradise Cleaner Incident"
 
 
 
+mission "Class Trip to Port"
+	minor
+	source
+		attributes "south"
+		not attributes "station"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`A group of children waves at you as you step off your ship, marveling at the spacecraft on display as their teachers try and fail to hold their attention. You can see the fading of their uniform colors from afar as their group walks past, yet none of them look one bit displeased. While you make your way into the spaceport itself, you wonder how many of them may eventually own their own spaceship and fly across the stars.`
+				decline
+
+
+
+mission "Sojourn Sunday Sales"
+	minor
+	source
+		government "Free Worlds"
+		attributes "urban"
+		not attributes "station"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`You blink in surprise as you enter the spaceport, watching throngs of people pushing their way through the human chains of queues wrapped around the commercial area. You scan your surroundings and notice advertisements upon advertisements of vacation locations, all at an absurdly high discount. Yet you cannot help but notice that to the younger you, it'd still be a price you'd never be willing to spend on such things.`
```

Comment:
Feels like this could use a little more elaboration. To me, the last sentence doesn't logically follow what's said before it super well.

Reply:
The idea here was that in the past as a factory worker, this kind of travel is still expensive. How that should be rephrased ATM, idk.

Reply:
```suggestion
			`You blink in surprise as you enter the spaceport, watching throngs of people pushing their way through the human chains of queues wrapped around the commercial area. You scan your surroundings and notice advertisements upon advertisements of vacation locations, all at an absurdly high discount. Yet despite the vastly reduced prices the advertised holidays still cost such large sums that you cannot help but think that a younger you could never have justified such expense.`
```

Reply:
```suggestion
			`You blink in surprise as you enter the spaceport, watching throngs of people pushing their way through the human chains of queues wrapped around the commercial area. You scan your surroundings and notice advertisements upon advertisements of vacation locations, all at an absurdly high discount. Yet despite their vastly reduced prices, the advertised holidays still cost such large sums that a younger you could never have justified such expense.`
```
That's a loooong sentence. I think adding that comma in and removing the `you cannot help but think` helps reduce wordiness. I think maybe rephrasing helped a lot more with understanding than elaboration - my apologies for not understanding that root issue.